include/grpc/impl/codegen/port_platform.h

```diff
@@ -187,6 +186,11 @@
 #else /* _LP64 */
 #define GPR_ARCH_32 1
 #endif /* _LP64 */
+#ifndef __GLIBC__ // musl
```

Comment:
If this is going to have a `#if`/`#else` anyway, it would be clearer to avoid a `#ifndef` clause and just use `#ifdef __GLIBC__`